routers/web/user/package.go

```diff
@@ -428,6 +433,11 @@ func PackageSettings(ctx *context.Context) {
 		ctx.ServerError("LoadHeaderCount", err)
 		return
 	}
+	_, err = shared_user.PrepareOrgHeader(ctx)
```

Comment:
Same as above: Should `PrepareOrgHeader` be called for a individual user (not an org)?